openc3/bin/openc3cli

```diff
@@ -33,6 +33,7 @@ require 'openc3/models/gem_model'
 require 'openc3/models/migration_model'
 require 'openc3/models/tool_model'
 require 'openc3/packets/packet_config'
+require 'openc3/script'
```

Comment:
Jason and I thought it could work, but I didn't test it.  I've put them all back, and made my new code do local requires too, so now there's no require 'openc3/script' at the top.